pkg/util/util.go

```diff
@@ -228,7 +224,7 @@ func IsValidProjectDir(path string, devfilePath string) error {
 // takes an absolute path prefixed with file:// and extracts it to a destination.
 // pathToUnzip specifies the path within the zip folder to extract
 // TODO(feloy) sync with devfile library?
-func GetAndExtractZip(zipURL string, destination string, pathToUnzip string, starterToken string) error {
+func GetAndExtractZip(zipURL string, destination string, pathToUnzip string, starterToken string, fsys filesystem.Filesystem) error {
```

Comment:
Can `fsys` be the first parameter of this function, just to be consistent with the other functions like `IsValidProjectDir`?

Reply:
I've changed the position of `fsys` in `IsValidProjectDir` just to be more consistent with other functions.